packages/imaplet-lwt/imaplet-lwt.0.1.13/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+name: "imaplet-lwt"
+version: "0.1.13"
+maintainer: "Gregory Tsipenyuk <gregtsip@gmail.com>"
+authors: "Gregory Tsipenyuk <gregtsip@gmail.com>"
+license: "MIT"
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+remove: [
+  ["ocamlfind" "remove" "imaplet"]
+  ["rm" "-f" "%{bin}%/imaplet*"]
```

Comment:
would it be possible to change this wildcard into an explicit set of rm commands?  It's a bit unsafe to just remove anything called imaplet